packages/web/.eslintrc.json

```
@@ -22,7 +22,7 @@
     "no-restricted-imports": [
       "error",
       {
-        "patterns": [".", "..", "./*", "../*"]
+        "patterns": [".", "..", "../*"]
```

```suggestion
        "patterns": [".", ".."]
```

Let's remove this as well, as it can be useful to import nearby components in the folder above.

---

@mattupham wait I think the recent commit removed the opposite of the suggestion?